content/learn/contribute/helping-out/creating-examples.md

```diff
@@ -29,7 +58,13 @@ Please adhere to the following guidelines when creating or updating an example.
 ### Organization
 
 1. Examples should live in an appropriate subfolder of `/examples`.
-2. Examples should be a single file if possible.
+   1. API examples live in `examples/api`, then subdivided by engine category.
+   2. Usage examples live in `examples/usage`, then subdivided by domain.
```

Comment:
Not sure the distinction between these two is clear to newcomers, maybe some better names can be found?

Reply:
Yeah, bikeshed welcome on this!

Reply:
I also think "patterns" works for smaller examples showcasing a single concept. Maybe even "snippets" or "parts". I also think that for these small examples, somehow linking them in the module-level docs of what they're shocasing would be neat, but that is probably out of scope for this. well, except maybe that api/pattern/snippet examples could be placed in folders matching the module hierarchy?